newIDE/app/src/ExportAndShare/Builds/BuildCard.js

```diff
@@ -117,7 +117,7 @@ const BUILD_DEFAULT_NAME_TIME_FORMAT = 'yyyy-MM-dd-HH-mm-ss';
 type Props = {|
   build: Build,
   game: Game,
-  onGameUpdated?: Game => void,
+  onGameUpdated?: () => Promise<void>,
```

Comment:
less optimal, but I've changed all places where a game is updated to let the top component refetch this game, rather than update it optimistically.
The reason is because we are sometimes updating slug & authors, via a different API call, which does not return the new state of the game, with cached values.
So it's safer to just say "it's been updated, load it again"

Reply:
Fair enough!